src/main/resources/db/migration/V2_1_0__test_data_load.sql

```diff
@@ -1,5 +1,5 @@
-INSERT INTO pharmacies(name) VALUES ('DrMax');
```

Comment:
to jest okej na potrzeby testowego projektu. Natomiast na przyszłość z tym co ja się spotkałem - Flywaya wykorzystujemy głównie do tworzenia bazy, kreowania struktur itd Nie dodajemy z jego poziomu rekordów. A to dlatego że uruchamiając apkę na produkcji, rekordy zostaną tam bezpośrednio wgrane.

Co innego gdybyśmy wgrywali jakieś enumy, tutaj natomaist są to zwykłe rekordy.

Także zostaw tak jak jest, ewentualnie pomyślimy jak można inaczej to rozwiązać. Idealnie by było też stworzyć kolekcje postmanową. To jutro pogadamy o tym